src/tests/dbus-tests/test_90_lsm.py

```diff
@@ -0,0 +1,42 @@
+import os
+import dbus
+import storagedtestcase
+
+
+class StoragedLsmLocalTestCase(storagedtestcase.StoragedTestCase):
+    """
+    Test suit for org.freedesktop.UDisks2.Drive.LsmLocal interface.
+    """
+    _LED_CONTROL_METHOD_NAMES = ["TurnFaultLEDOn", "TurnFaultLEDOff",
+                                 "TurnIdentLEDOn", "TurnIdentLEDOff"]
+
+    def _get_dbus_drv_obj(self, dbus_blk_obj):
+        obj_path = self.get_property(dbus_blk_obj, '.Block', 'Drive')
```

Comment:
`StoragedTestCase.get_property` method has changed in 06f9d4f12e89fa79befb5f82ce810b5d44f41289 You should use `get_property_raw` instead.